sno/workers/README.md

```diff
@@ -0,0 +1,215 @@
+# **Running sno with extra workers**
+
+This document provides information on how to deploy a SNO and add extra workers to it, and then study behaviour when losing some of the workers or the initial node
+
+Note: appendix contains kcli commands used to speed up the workflow
+
+## **Environment**
+
+* We run all the deployments using libvirt vms, although the procedure would be the same for baremetal nodes.
+
+## **Initial deployment**
+
+1. Deploying SNO
+
+We leverage bootstrap in place procedure to generate a valid ignition file that is then embedded using coreos-installer
+
+We start with the following install config where:
+
+- we specify platform as None
+- we indicate in the bootstrap in place the disk to use for installation
+- we set network type to OVN
+
+```
+apiVersion: v1
+baseDomain: karmalabs.com
+compute:
+- name: worker
+  replicas: 0
+controlPlane:
+  name: master
+  replicas: 1
+metadata:
+  name: mycluster
+networking:
+  networkType: OVNKubernetes
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+  clusterNetwork:
+  - cidr: 10.132.0.0/14
+    hostPrefix: 23
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+BootstrapInPlace:
+  InstallationDisk: /dev/vda
+pullSecret: 'XXX'
+sshKey: |
+  ssh-rsa ZZZ
+```
+
+With this install config, we then generate an ignition to use for SNO
+
+```
+mkdir ocp
+cp install-config.yaml ocp
+openshift-install --dir=ocp create single-node-ignition-config
+```
+
+This will generate a file named bootstrap-in-place-for-live-iso.ign that we can embed into any valid rhcos live iso, such as the [latest one](https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/latest/rhcos-live.x86_64.iso) using the following invocation:
+
+```
+coreos-installer iso ignition embed -fi iso.ign bootstrap-in-place-for-live-iso.ign -o bip.iso rhcos-live.x86_64.iso
+```
+
+Once we boot a given node with this generated bip.iso, after a while, we should see an output similar to the following one:
+
+```
+# oc get node
+NAME                          STATUS   ROLES           AGE     VERSION
+mycluster-sno.karmalabs.com   Ready    master,worker   176m    v1.22.3+b93fd35
+```
+
+2. Adding additional workers
+
+Although ignition for workers is not generated when running openshift-install as in the previous section, this can be retrieved using curl
```

Comment:
```suggestion
Although ignition for workers is not generated when running openshift-install as in the previous section, this can be retrieved using curl:
```